### ISSUE TITLE
[Go] Implement new Concept Exercise: structs

### ISSUE THREAD

**Issue by tehsphinx** [exercism/v3#2029]
This issue describes how to implement the `structs` concept exercise for the Go track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student what `structs` in Go are and how to use them.

## Learning objectives

- Know what structs are
- How to define a struct
- How to instantiate a struct (empty and filled)
- How to read/write fields of a struct
- Hint on: Structs are the classes of Go

Maybe suggest to do methods exercise next as it is a direct continuation of structs and makes the topic complete.

## Out of scope

- methods
- pointers
- interfaces

## Concepts

- `structs`

## Prerequisites

- `basics`

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

**Comment by oanaOM** [comment]
This concept has been implemented so I'll close this.
